### ISSUE TITLE
Assignment options missing.

### ISSUE THREAD

**Issue by smekras** [mozilla/multi-account-containers#569]
Using Firefox 53.0.3 on FC25, with Containers 2.3.0 enabled, but I can't see any option related to assignment, or "always open in this container" or anything similar. I tried disabling and reenabling Containers to no avail.

Am I missing something? Is there some flag I need to enable?

**Comment by jonathanKingston** [comment]
@smekras are you opening a container first then using the URL? The menu item is on right click within the tab. (we are opening up this to better UX soon)

If not do you set your history to always expire and treat cookies as sessions? (We call this always private mode).

**Comment by smekras** [comment]
I was opening the container first. Didn't seem to make a difference though.

Now I put Firefox on always-private and the addon seems to have stopped working entirely.

**Comment by jonathanKingston** [comment]
@smekras yeah we don't support that mode yet.

Could you post a copy of what is in about:support?

The menu item won't appear sometimes when the menu is loading and also for about: pages for example.

**Comment by smekras** [comment]
This is pretty much it. Now the addon allows me to "Open Link in New Container Tab" but not for bookmarks, and I still don't get an option to assign a specific site to a specific container.


Application Basics
------------------

Name: Firefox
Version: 53.0.3
Build ID: 20170526184742
User Agent: Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:53.0) Gecko/20100101 Firefox/53.0
OS: Linux 4.11.3-200.fc25.x86_64
Multiprocess Windows: 1/1 (Enabled by user)
Safe Mode: false

Extensions
----------

Name: better-youtube-subs
Version: 1.19.1
Enabled: true
ID: jid1-PmCaAQKMFABjHg@jetpack

Name: Containers Experiment
Version: 2.3.0
Enabled: true
ID: @testpilot-containers

Name: GNOME Shell integration
Version: 8.2.1
Enabled: true
ID: chrome-gnome-shell@gnome.org

Name: Min Vid
Version: 0.3.7
Enabled: true
ID: @min-vid

Name: Multi-process staged rollout
Version: 1.15
Enabled: true
ID: e10srollout@mozilla.org

Name: Pocket
Version: 1.0.5
Enabled: true
ID: firefox@getpocket.com

Name: Test Pilot
Version: 1.1.2-dev-f3fe9bf
Enabled: true
ID: @testpilot-addon

Name: uBlock Origin
Version: 1.12.4
Enabled: true
ID: uBlock0@raymondhill.net

Name: Web Compat
Version: 1.0
Enabled: true
ID: webcompat@mozilla.org

Graphics
--------

Features
Compositing: OpenGL
Asynchronous Pan/Zoom: wheel input enabled
WebGL Renderer: X.Org -- Gallium 0.4 on AMD POLARIS11 (DRM 3.10.0 / 4.11.3-200.fc25.x86_64, LLVM 3.9.1)
WebGL2 Renderer: X.Org -- Gallium 0.4 on AMD POLARIS11 (DRM 3.10.0 / 4.11.3-200.fc25.x86_64, LLVM 3.9.1)
Audio Backend: pulse
GPU #1
Active: Yes
Description: X.Org -- Gallium 0.4 on AMD POLARIS11 (DRM 3.10.0 / 4.11.3-200.fc25.x86_64, LLVM 3.9.1)
Vendor ID: X.Org
Device ID: Gallium 0.4 on AMD POLARIS11 (DRM 3.10.0 / 4.11.3-200.fc25.x86_64, LLVM 3.9.1)
Driver Version: 3.0 Mesa 17.0.5

Diagnostics
AzureCanvasAccelerated: 0
AzureCanvasBackend: skia
AzureContentBackend: skia
AzureFallbackCanvasBackend: none
CairoUseXRender: 0
Decision Log
HW_COMPOSITING:
blocked by default: Acceleration blocked by platform
force_enabled by user: Force-enabled by pref
OPENGL_COMPOSITING:
force_enabled by user: Force-enabled by pref




Important Modified Preferences
------------------------------

accessibility.typeaheadfind.flashBar: 0
browser.cache.disk.capacity: 358400
browser.cache.disk.filesystem_reported: 1
browser.cache.disk.smart_size.first_run: false
browser.cache.frecency_experiment: 3
browser.download.importedFromSqlite: true
browser.places.smartBookmarksVersion: 8
browser.sessionstore.upgradeBackup.latestBuildID: 20170526184742
browser.startup.homepage_override.buildID: 20170526184742
browser.startup.homepage_override.mstone: 53.0.3
browser.tabs.remote.autostart: true
browser.urlbar.daysBeforeHidingSuggestionsPrompt: 0
browser.urlbar.lastSuggestionsPromptDate: 20170217
dom.apps.lastUpdate.buildID: 20160919122928
dom.apps.lastUpdate.mstone: 49.0
dom.apps.reset-permissions: true
dom.push.userAgentID: 44a1486a18ed4c63aead4b6e86a55bfb
extensions.lastAppVersion: 53.0.3
font.internaluseonly.changed: true
layers.acceleration.force-enabled: true
media.gmp-gmpopenh264.abi: x86_64-gcc3
media.gmp-gmpopenh264.lastUpdate: 1487354520
media.gmp-gmpopenh264.version: 1.6
media.gmp-manager.buildID: 20170526184742
media.gmp-manager.lastCheck: 1496520107
media.gmp.storage.version.observed: 1
media.webrtc.debug.log_file: /tmp/WebRTC.log
network.cookie.prefsMigrated: true
network.dns.disablePrefetch: true
network.http.speculative-parallel-limit: 0
network.predictor.cleaned-up: true
network.prefetch-next: false
places.database.lastMaintenance: 1496209636
places.history.expiration.transient_current_max_pages: 132492
plugin.disable_full_page_plugin_for_types: application/pdf
plugin.importedState: true
plugin.state.libgnome-shell-browser-plugin: 2
print.print_bgcolor: false
print.print_bgimages: false
print.print_duplex: 0
print.print_evenpages: true
print.print_in_color: true
print.print_margin_bottom: 0.5
print.print_margin_left: 0.5
print.print_margin_right: 0.5
print.print_margin_top: 0.5
print.print_oddpages: true
print.print_orientation: 0
print.print_page_delay: 50
print.print_paper_data: 0
print.print_paper_height: 11.00
print.print_paper_name: na_letter
print.print_paper_size_unit: 0
print.print_paper_width: 8.50
print.print_scaling: 1.00
print.print_shrink_to_fit: true
print.print_to_file: false
print.print_unwriteable_margin_bottom: 56
print.print_unwriteable_margin_left: 25
print.print_unwriteable_margin_right: 25
print.print_unwriteable_margin_top: 25
privacy.history.custom: true
privacy.usercontext.about_newtab_segregation.enabled: true
privacy.userContext.enabled: true
services.sync.declinedEngines: prefs,addons
services.sync.engine.addons: false
services.sync.engine.prefs: false
services.sync.lastPing: 1496462752
services.sync.lastSync: Sun Jun 04 2017 03:08:05 GMT+0300 (EEST)
services.sync.numClients: 2
storage.vacuum.last.index: 1
storage.vacuum.last.places.sqlite: 1495258559

Important Locked Preferences
----------------------------

Places Database
---------------

JavaScript
----------

Incremental GC: true

Accessibility
-------------

Activated: false
Prevent Accessibility: 0

Library Versions
----------------

NSPR
Expected minimum version: 4.14
Version in use: 4.14

NSS
Expected minimum version: 3.30.2
Version in use: 3.30.2

NSSSMIME
Expected minimum version: 3.30.2
Version in use: 3.30.2

NSSSSL
Expected minimum version: 3.30.2
Version in use: 3.30.2

NSSUTIL
Expected minimum version: 3.30.2
Version in use: 3.30.2

Experimental Features
---------------------

Sandbox
-------

Seccomp-BPF (System Call Filtering): true
Seccomp Thread Synchronization: true
User Namespaces: true
Media Plugin Sandboxing: true

**Comment by jonathanKingston** [comment]
Bookmarks doesn't work currently either, there is a feature request for that.

So the steps to get the item is you need to be in a website (about: urls are not available) and also in a container tab. Is that what you are trying?

**Comment by Edicatad** [comment]
I have this same issue, it shows up intermittently for me. I open a website in a container, then right click on the background somewhere, and no container options show up.  I just tried it again on a website I knew it didn't work on and after being on other tabs for a while it worked.  I presume it's not related to the website loading in completely so I have no idea what actually causes it.

As an alternative method, is there any way to just specify domains in a config file somewhere to get around this for now?

**Comment by jonathanKingston** [comment]
I'm going to call this a duplicate of #855 which has clear steps to reproduce this and with the latest code.

Thanks for raising please follow there.
